src/Modules/Order/orderService.js

```diff
@@ -0,0 +1,42 @@
+const DbService = require("../../Service/DbService");
+const serviceHandler = require("../../Utils/serviceHandler");
+const CustomError = require("../../Errors/CustomError");
+const OrderModel = require("./orderModel");
+
+const model = new DbService(OrderModel);
+
+const orderService = {
+  create: serviceHandler(async (data) => {
+    return await model.save(data);
+  }),
+
+  getAll: serviceHandler(async (data) => {
+    const query = { isDelete: false };
+    const savedData = await model.getAllDocuments(query, data);
+    const totalCount = await model.totalCounts({ isDelete: false });
+    return { savedData, totalCount };
+  }),
```

Comment:
in getAll and getById, do not return order directly, populate customerId and productId before returning. 
Try using chatgpt or google to learn how to populate a field. 
Do same for update() method as well